src/main/resources/messages_de.properties

```diff
@@ -0,0 +1,52 @@
+widget.meta.title=Mit BundesIdent online ausweisen
+widget.headline-title-top=Mit BundesIdent
+widget.headline-title-bottom=online ausweisen
+widget.headline-image-alt=Drei Ausweisdokumente: Personalausweis, \
+  elektronischer Aufenthaltstitel und die eID-Karte für Bürgerinnen und Bürger \
+  der Europäischen Union und des Europäischen Wirtschaftsraums
+widget.app-icon-image-alt=App-Symbol mit weißem Bundes-Adler auf blauem Grund
+widget.step-one-alt=Schritt eins
+widget.step-two-alt=Schritt zwei
+widget.download-title=BundesIdent installieren
+widget.app-store-alt=Aus dem Apple App Store herunterladen
+widget.play-store-alt=Aus dem Google Play Store herunterladen
+widget.identification-button=Mit BundesIdent ausweisen
+widget.identification-button-title=BundesIdent App zum online Ausweisen öffnen
+widget.data-privacy-button=Datenschutzerklärung
+widget.app-store-url=https://apps.apple.com/de/app/bundesident/id1635758944
+widget.play-store-url=https://play.google.com/store/apps/details?id=de.\
+  digitalService.useID
+widget.data-privacy-url=https://digitalservice.bund.\
+  de/datenschutzerklaerung-bundesident
+
+widget.incompatible.headline-title=Ihr Smartphone erfüllt leider nicht die \
+  Voraussetzungen der BundesIdent App.
+widget.incompatible.headline-image-alt=Zeichnung eines Smartphones mit \
+  traurigem Gesicht und App-Symbol mit weißem Bundes-Adler auf blauem Grund
+widget.incompatible.requirement-title=Mindestvoraussetzungen:
+widget.incompatible.android-title=Android
+widget.incompatible.android-requirement=Android 9 »Pie« und ein NFC-Sensor.
+widget.incompatible.iphone-title=iPhone
+widget.incompatible.iphone-requirement=iOS 15 & iPhone 7.
+
+widget.fallback.error.text=Stellen Sie sicher, dass die BundesIdent App \
+  installiert ist und versuchen Sie es erneut.
+
+error.default.icon-alt=Ausrufezeichen auf rotem Grund
+error.default.title=Es ist ein Fehler aufgetreten.
+error.default.text=Bitte starten Sie die Identifizierung erneut. Sollte erneut \
+  ein Fehler auftreten, freuen wir uns über eine kurze Rückmeldung an:
+error.default.report-email=hilfe@bundesident.de
+error.default.report-subject=Fehlermeldung mit Status {0}
+error.default.report-body=Liebes BundesIdent-Team, \n\
+  \n\
+  \ ich habe erneut eine Fehlermeldung mit dem Status {0} erhalten. \n\
+  \n\
+  \ Ich wollte mich bei folgendem Diensteanbieter identifizieren: \n\
+  \ Der Fehler trat auf, nachdem ich: \n\
+  \ Ich benutze folgendes Gerät: \n\
+  \ Das Betriebssystem meines Gerätes: \n\
+  \ Ich benutze folgenden Browser: \n\
+  \ Die Version von meinem Browser: \n\
+  \n\
+  \ Mit freundlichen Grüßen
```

Comment:
I guess this is the way to write multi-line in `.properties` ? I searched the alternative for YAML, because with YAML multi-line is more expressive, requires no backward slash, but I could not find the solution 😓

Reply:
Unfortunately, YAML is currently not supported for message files. I actually had the whole String in one line but `spotless` added the newlines with backslash.